troupon/staticfiles/scss/partials/_resets.scss

```diff
@@ -0,0 +1,59 @@
+
+/* ==========================================================================
+Resets (overrides for Bootstrap and browser-defaults)
+========================================================================== */
+
+body {
+	appearance: none;
+}
+
+h2, h3, h4 {
+	margin: 0px 0px 10px 0px;
```

Comment:
Line should be indented with spaces, not tabs<br>Shorthand form for property `margin` should be written more concisely as `0px 0px 10px` instead of `0px 0px 10px 0px`<br>Each selector in a comma sequence should be on its own line<br>`0px` should be written without units as `0`